src/pod.ts

```diff
@@ -73,6 +75,13 @@ export class Pod {
     };
     this.cache = new Cache(this);
 
+    // Register built-in plugins before the amagaki.js config to be consistent with
+    // external plugin hooks and allow external plugins to work with the built-in
+    // plugins.
+    for (const BuiltInPlugin of BUILT_IN_PLUGINS) {
```

Comment:
Ultra-tiny organizational nit but I think the list of built-in plugins should be in `pod.ts` and not in `plugins.ts`, e.g. `Pod.DefaultPlugins` (like we have the `DefaultXXXX` on the `Pod` object).

Reply:
Ok, I can move it there. I'm not going to call it `DefaultPlugins` though since it is not really passed in atm to replace it. If we change it later to allow passing in a set of plugins than we can rename it.